docs/rpc_nodes.md

```diff
@@ -25,14 +25,14 @@ values={[
 |------------------|--------------|------------------------------------------|---------------------------------------------------------------------------------|
 | ECAD Labs        | Mainnet      | https://mainnet.api.tez.ie               | [Check](https://mainnet.api.tez.ie/chains/main/blocks/head/header)              |
 | ECAD Labs        | Ghostnet     | https://ghostnet.ecadinfra.com           | [Check](https://ghostnet.ecadinfra.com/chains/main/blocks/head/header)          |
+| ECAD Labs        | Nairobinet   | https://nairobinet.ecadinfra.com         | [Check](https://nairobinet.ecadinfra.com/chains/main/blocks/head/header)        |
 | ECAD Labs        | Mumbainet    | https://mumbainet.ecadinfra.com          | [Check](https://mumbainet.ecadinfra.com/chains/main/blocks/head/header)         |
-| ECAD Labs        | Limanet      | https://limanet.ecadinfra.com            | [Check](https://limanet.ecadinfra.com/chains/main/blocks/head/header)           |
 | SmartPy          | Mainnet      | https://mainnet.smartpy.io               | [Check](https://mainnet.smartpy.io/chains/main/blocks/head/header)              |
 | SmartPy          | Ghostnet     | https://ghostnet.smartpy.io              | [Check](https://ghostnet.smartpy.io/chains/main/blocks/head/header)             |
-| SmartPy          | Limanet      | https://limanet.smartpy.io/              | [Check](https://limanet.smartpy.io/chains/main/blocks/head/header)              |
```

Comment:
quick question, is SmartPy not running temp testnet anymore?

Reply:
I think the url might just be dead since it's Limanet? But I don't know if smartpy still hosts their own testnets or not